src/contracts/Protocol/ClaimController.sol

```diff
@@ -6,13 +6,22 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 // Interfaces
 import "./interfaces/IClaimController.sol";
 
+// Enums
+import "./Enums/ClaimEnum.sol";
+
+// Structs
+import "./Structs/AnswerStruct.sol";
+
 // Modifiers
 import "./modifiers/OnlyAPI.sol";
 
 contract ClaimController is Ownable, IClaimController, OnlyApi {
     /// @notice Keeps track of claim limits per question
     mapping(uint256 => uint256) public claimLimits;
 
+    /// @notice Keeps track of claim counts per question
+    mapping(uint256 => uint256) public claimCounts;
```

Comment:
wouldn't

claimCounts[X] == answers[x].length

?  Do we need a second mapping for this?